index.html

```diff
@@ -5,63 +5,259 @@
-                 don't hestiate to contact me.</p>
+        <!-- 4th -->
+        <div id="work_4">
+            <div class="a">
+                <img src="images/header/second.png" alt="tonic">
+            </div>
+            <div class="b">Multi-Post Stories</div>
+            <div class="c">
+                    <div class="canopy">CANOPY</div>
+                    <div class="dot">•</div>
+                    <div class="job">Back End Dev</div>
+                    <div class="dot">•</div>
+                    <div class="yrs">2015</div>   
+            </div>
+            <div class="d">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="e">
+                <button class="one">html</button>
+                <button class="two">css</button>
+                <button class="three">javaScript</button>
+            </div>
+            <div class="f">
+                <button>See Project</button>
+            </div>
         </div>
-        <div class="socialMedia">
-            <h3>
-                LET'S CONNECT
-            </h3>
-        </div>
-        <div class="socialMedia_logo">
-            <img src="images/header/logo.png" alt="icon">
-        </div>
-    </section>
-   </main>
+        <!-- about myself -->
+        <div id="about">
+            <div id="box1">
+                <h2>About <br> Myself</h2>
+                <div class="intro">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me
+                </div>
+                <div class="connect">LET'S CONNECT</div>
+                <div class="link">
+                    <ul>
+                        <li>
+                            <a class="Github" href="">
+                                <img src="images/header/github.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="LinkedIn" href="">
+                                <img src="images/header/linkedin.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="Ok" href="">
+                                <img src="images/header/ok.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="Twitter" href="">
+                                <img src="images/header/twiter.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="M" href="">
+                                <img src="images/header/m.png" alt="">
+                            </a>
+                        </li>
```

Comment:
- [ ] Please, check if all the `img` tags have an `alt` attribute with some meaningful text describing the images.